bin/hledger-prices.hs

```diff
@@ -19,9 +20,10 @@ import Hledger.Cli
 ------------------------------------------------------------------------------
 cmdmode = hledgerCommandMode
   [here| prices
-Print all prices from the journal.
+Print all market prices from the journal.
   |]
-  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings instead of market prices"]
+  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings"
+  ,flagNone ["inverted-costs"] (setboolopt "inverted-costs") "print transaction inverted prices from postings also"]
   [generalflagsgroup1]
```

Comment:
> But, sorry, I don't follow what you mean with the second part ("consider that I'm abroad...") ?

I referred to a different kind of transactions where you pay in supermarket from an account of different currency. I.e. when withdraw from card is in one currency and there is several expense postings in other one.
In that case I would prefer to have currency exchange rate only near to withdrawal posting which had to be specified in an "inverted" form and thus `--costs` is not enough to catch that.

> What's your opinion on consistently using "price" and "transaction price" (or something better) rather than "cost" ?

I still don't feel to fully understand wording of `--cost` and `--value` to effectively contribute to decision about dropping mention of "cost".

>    -B --cost               convert amounts to their cost at transaction time
>                           (using the transaction price, if any)
>   -V --value              convert amounts to their market value on the report
>                           end date (using the most recent applicable market
>                           price, if any)
> 

And the fact that I'm not an English native speaking person this makes it even hard for me to be sure in how to distinguish "cost" and "price". For me difference is that "cost" usually refers to actually committed expenses in past and "price" usually refers to some indicator rather than real expense. I.e. "cost" may differ from "price" and that's why I thought `--cost` used to refer to application of past "price" while `--value` were used to estimate current "value" according to recent "price" that can be used to sell something.

> As the world's most knowledgeable user of this command, may I ask you: in your real world usage, when/why do you use this command, the --cost flag, and the --inverted-cost flag ?

Maybe I'm the only one who uses this "plugin" :) . My use cases:
1. See how rate of my main currency changes to have a rough estimation of future conversion rates. I.e. look into other currencies in term of my main currency. `P 2017-01-01 EUR $1.07`
1. Copy-paste some rates from prev. transaction to set market prices for reports with options like `-B`.
```
2017/1/1 My Bank | buy some currency for upcoming trip
  assets:bank  $-108
  assets:cash  100 EUR @ $1.07  ; reported by `--inverted-cost`
  expenses:bank  $1

2017/2/1 Supermarket | spend some money from my
  assets:bank  $-107 @@ 100 EUR  ; according to bank report and receipt. re
  expenses:grocery  50 EUR
  expenses:snacks  50 EUR
```
Effectively
```
zsh% ./bin/hledger-prices --costs
P 2017-01-01 EUR $1.07
P 2017-02-01 $ 0.935 EUR
zsh% ./bin/hledger-prices --inverted-costs
P 2017-01-01 $ 0.935 EUR
P 2017-02-01 EUR $1.070
```
Mostly I use `--costs` to get rates with which I've got exchange. And with `--costs --inverted-costs` I get source for picking prices to put into my journal.